cypress/integration/navigationFlows/homePageLeftSidebar.spec.js

```diff
@@ -1,35 +0,0 @@
-describe('Home Page Left Sidebar', () => {
```

Comment:
I removed this Cypress spec file because the only thing it was testing was the `More...` button functionality in the Navigation Sidebar; in this PR, I am deprecating that button.